src/main/java/org/eclipse/yasson/internal/Marshaller.java

```diff
@@ -135,33 +143,72 @@ public void marshallWithoutClose(Object object, JsonGenerator jsonGenerator) {
     @SuppressWarnings("unchecked")
     public <T> void serializeRoot(T root, JsonGenerator generator) {
         if (root == null) {
-            getJsonbContext().getConfigProperties().getNullSerializer().serialize(null, generator, this);
+            jsonbContext.getConfigProperties().getNullSerializer().serialize(null, generator, this);
             return;
         }
         final JsonbSerializer<T> rootSerializer = (JsonbSerializer<T>) getRootSerializer(root.getClass());
-        if (getJsonbContext().getConfigProperties().isStrictIJson()
+        if (jsonbContext.getConfigProperties().isStrictIJson()
                 && rootSerializer instanceof AbstractValueTypeSerializer) {
             throw new JsonbException(Messages.getMessage(MessageKeys.IJSON_ENABLED_SINGLE_VALUE));
         }
         rootSerializer.serialize(root, generator, this);
     }
 
     JsonbSerializer<?> getRootSerializer(Class<?> rootClazz) {
-        final ContainerSerializerProvider serializerProvider = getMappingContext().getSerializerProvider(rootClazz);
+        final ContainerSerializerProvider serializerProvider = jsonbContext.getMappingContext().getSerializerProvider(rootClazz);
         if (serializerProvider != null) {
             return serializerProvider
                     .provideSerializer(new JsonbPropertyInfo()
                                                .withRuntimeType(runtimeType));
         }
-        SerializerBuilder serializerBuilder = new SerializerBuilder(getJsonbContext())
+        SerializerBuilder serializerBuilder = new SerializerBuilder(jsonbContext)
                 .withObjectClass(rootClazz)
                 .withType(runtimeType);
 
         if (!DefaultSerializers.getInstance().isKnownType(rootClazz)) {
-            ClassModel classModel = getMappingContext().getOrCreateClassModel(rootClazz);
+            ClassModel classModel = jsonbContext.getMappingContext().getOrCreateClassModel(rootClazz);
             serializerBuilder.withCustomization(classModel.getClassCustomization());
         }
         return serializerBuilder.build();
     }
     
+    /**
+     * Jsonb context.
+     *
+     * @return jsonb context
+     */
+    public JsonbContext getJsonbContext() {
```

Comment:
perhaps we can just remove this getter method entirely? Since the `jsonbContext` field is final there is no difference in a public getter vs. a public final method (although we probably only need package-protected)

Reply:
That would make sense... a getter method is completely pointless in this case... I tried to do it, but Checkstyle gave me this warning: "Variable 'jsonbContext' must be private and have accessor methods"